internal/route/route.go

```diff
@@ -13,6 +13,6 @@ import (
 
 // GrpcRoute func
 func GrpcRoute(grpcServer *grpc.Server, db *sql.DB, log *log.Logger, cache *redis.Cache) {
-	quizServer := quizDomain.QuizService{Db: db, Cache: cache}
+	quizServer := quizDomain.QuizService{Db: db, Cache: cache, Log: log}
```

Comment:
suggestion: use slog package for better logging printing https://go.dev/blog/slog